### PR TITLE
Upgrade minimum supported k8s version to v1.12

### LINKED ISSUE #7720: flake test: WithError(Failed to setup certs)=certificate symlinks
as seen here: 
https://storage.googleapis.com/minikube-builds/logs/7705/c44655c/Docker_Linux.html#fail_TestStartStop%2fgroup%2fold-docker

```

	I0416 07:59:55.037956   15523 ssh_runner.go:148] Run: sudo /bin/bash -c "test -s /usr/share/ca-certificates/minikubeCA.pem && ln -fs /usr/share/ca-certificates/minikubeCA.pem /etc/ssl/certs/minikubeCA.pem"
	I0416 07:59:55.050765   15523 ssh_runner.go:148] Run: ls -la /usr/share/ca-certificates/minikubeCA.pem
	I0416 07:59:55.056011   15523 certs.go:382] hashing: -rw-r--r-- 1 root root 1066 Apr 16 14:45 /usr/share/ca-certificates/minikubeCA.pem
	I0416 07:59:55.056098   15523 ssh_runner.go:148] Run: openssl x509 -hash -noout -in /usr/share/ca-certificates/minikubeCA.pem
	I0416 07:59:55.063764   15523 ssh_runner.go:148] Run: sudo /bin/bash -c "test -L /etc/ssl/certs/b5213941.0 || ln -fs /etc/ssl/certs/minikubeCA.pem /etc/ssl/certs/b5213941.0"
	I0416 07:59:55.076632   15523 ssh_runner.go:148] Run: sudo /bin/bash -c "test -s /usr/share/ca-certificates/6700.pem && ln -fs /usr/share/ca-certificates/6700.pem /etc/ssl/certs/6700.pem"
	I0416 07:59:55.078009   15523 ssh_runner.go:148] Run: sudo systemctl start kubelet
	I0416 07:59:55.114008   15523 exit.go:58] WithError(Failed to setup certs)=certificate symlinks: create symlink for /usr/share/ca-certificates/6700.pem: sudo /bin/bash -c "test -s /usr/share/ca-certificates/6700.pem && ln -fs /usr/share/ca-certificates/6700.pem /etc/ssl/certs/6700.pem": Process exited with status 1
	stdout:
	
	stderr:
	 called from:
	goroutine 346 [running]:
	runtime/debug.Stack(0x0, 0x0, 0x20fb)
		/usr/local/go/src/runtime/debug/stack.go:24 +0x9d
	k8s.io/minikube/pkg/minikube/exit.WithError(0x1ac8e78, 0x15, 0x1d7d240, 0xc0005bada0)
		/app/pkg/minikube/exit/exit.go:58 +0x34
	k8s.io/minikube/pkg/minikube/node.setupKubeAdm.func2(0x1de8d40, 0xc0004e3290, 0xc000386300, 0x0, 0x0, 0xc00053e6a0, 0xa, 0x20fb, 0xc000622340, 0x8, ...)
		/app/pkg/minikube/node/start.go:276 +0x155
	created by k8s.io/minikube/pkg/minikube/node.setupKubeAdm
		/app/pkg/minikube/node/start.go:274 +0x513
	W0416 07:59:55.114274   15523 out.go:201] Failed to setup certs: certificate symlinks: create symlink for /usr/share/ca-certificates/6700.pem: sudo /bin/bash -c "test -s /usr/share/ca-certificates/6700.pem && ln -fs /usr/share/ca-certificates/6700.pem /etc/ssl/certs/6700.pem": Process exited with status 1
	stdout:
	
	stderr:
	* 
	X Failed to setup certs: certificate symlinks: create symlink for /usr/share/ca-certificates/6700.pem: sudo /bin/bash -c "test -s /usr/share/ca-certificates/6700.pem && ln -fs /usr/share/ca-certificates/6700.pem /etc/ssl/certs/6700.pem": Process exited with status 1
	stdout:
	
	stderr:
	
	* 
	* minikube is exiting due to an error. If the above message is not useful, open an issue:
	  - https://github.com/kubernetes/minikube/issues/new/choose
** /stderr **
start_stop_delete_test.go:132: failed to start minikube post-stop. args "out/minikube-linux-amd64 start -p old-docker-20200416T075527-6700 --memory=2200 --alsologtostderr -v=3 --wait=true --kvm-network=default --kvm-qemu-uri=qemu:///system --disable-driver-mounts --keep-context=false --container-runtime=docker --driver=docker  --kubernetes-version=v1.11.10": exit status 70

```

To run locally:
```
make integration -e TEST_ARGS="-test.run TestStartStop/group/old-docker --profile=minikube --cleanup=false --minikube-start-args="--driver=docker""
```

### PR DESCRIPTION
**Pull request by priyawadhwa**
This fixes #7720 #7541 

First, I reverted code to run update cluster and setupCerts in sequence to avoid the race condition mentioned here: https://github.com/kubernetes/minikube/issues/7541#issuecomment-612066460

The TestStartStop/group/old-docker test was also failing because it was running k8s v1.11.10 which relies on kubeadm template version v1alpha1. v1alpha1 doesn't allow us to set parameters to prevent a DiskPressure error, and so this test was failing because the node was constantly under DiskPressure.

Upgrading to v1.12.0 as minimum supported version, which uses kubeadm template v1alpha3, fixes this issue, as it does allow us to set the necessary parameters.

### CODE REVIEW

File: test/integration/start_stop_delete_test.go [hunk 1]
```diff
@@ -174,7 +174,7 @@ func TestStartStop(t *testing.T) {
 	})
 }
 
-func TestStartStopWithPreload(t *testing.T) {
+func TestPreloadStartStop(t *testing.T) {
```
Comment (medyagh) [inline test/integration/start_stop_delete_test.go:177]: how about renaming it to TestPreload, how about moving it to its own file , this file is getting too big.
preload_test.go ?

File: pkg/minikube/node/start.go [hunk 2]
```diff
@@ -261,24 +261,16 @@ func setupKubeAdm(mAPI libmachine.API, cfg config.ClusterConfig, n config.Node,
...
-		if err := bs.SetupCerts(cfg.KubernetesConfig, n); err != nil {
-			exit.WithError("Failed to setup certs", err)
-		}
-		parallel.Done()
-	}()
+	if err := bs.UpdateCluster(cfg); err != nil {
+		exit.WithError("Failed to update cluster", err)
+	}
+
+	if err := bs.SetupCerts(cfg.KubernetesConfig, n); err != nil {
```
Comment (medyagh) [inline pkg/minikube/node/start.go:270]: how about calling bs.SetupCerts inside bs.UpdateCluster  and then there decide a good parallization strattegy inside update cluter?

Comment (priyawadhwa) [reply pkg/minikube/node/start.go:270]: I'll probably take a look at that in another PR since this one is getting big

**Pull request merged**